README.md

```diff
@@ -132,6 +138,16 @@ qcSpy
 qcCheckerLauncher -c file:///absolute/path/to/example-default.ini -n checker_0
 ```
 
+### QCG (Web QC GUI)
+
+To run :
+```
+alienv enter qcg/latest-o2-dataflow
```

Comment:
A minor inconsequence, that we could fix: in line 18 you use `o2` defaults in the build command for qcg and here you enter environment with `latest-o2-dataflow` defaults.

Reply:
valid point, fixed